apps/prairielearn/src/server.ts

```diff
@@ -2409,7 +2409,9 @@ if (esMain(import.meta) && config.startServer) {
     logger.error('Error initializing PrairieLearn server:', err);
     throw err;
   }
-  logger.info('PrairieLearn server ready, press Control-C to quit');
+  logger.info('PrairieLearn server ready');
+  logger.info(`To attempt a clean shutdown run \`kill ${process.pid}\` from another process`);
```

Comment:
This isn't an acceptable level of complexity to show to everyone that's using PL for local development, who would now have to figure out a) what a clean shutdown is, b) if they actually want one, and c) how to get another process to run `kill` from.

It'd be more appropriate to document `SIGTERM` in the production deployment documentation.